### DISCUSSION TITLE
urgent, can't deploy, accidentally switched off premium plan

### DISCUSSION THREAD

**Discussion by ghost** [vercel/vercel#4221] [General] [answered]
We've been on the 'Premium' plan now for several years. We're in the process of migrating off Now 1.0, but we're not there yet. We switched off 'Premium' yesterday (April 30th), and now we can't deploy. We are getting an error that new deploys for Now 1.0 are disabled. We had received a message on April 24th that new Now 1.0 deploys were being stopped on May 1st.

**We can't make changes to our production environment. Please help urgently.**

**Comment by coetry** [comment, marked as answer]
Hi @robertspizzarottisoft ! If you ever have an urgent request, please reach out to support@vercel.com and we will take a look at your case since that will always take priority from GitHub Discussions. 

We've already sent out emails that highlight the deprecation timeline of v1 and this date for disabling new v1 deployments has been explicitly mentioned. I would highly recommend migrating your application to another container provider to avoid the sudden service halt on **August 7th**.
